src/components/@helpers/dropdown/Dropdown.tsx

```diff
@@ -42,11 +42,11 @@ export const Dropdown: React.FunctionComponent<DropdownProps> = ({}) => {
               return (
                 <>
                   <button 
-                    key={option.name}
+                    key={option.title}
```

Comment:
Nice. I think we can make the key more unique. Let's try the position of the option in the array returned @KBandipo